src/shared/components/image-previewer/image-previewer.component.ts

```diff
@@ -10,14 +10,18 @@ export class ImagePreviewerComponent {
   @Output() files: EventEmitter<FileReader> = new EventEmitter();
   @ViewChild('images') images: ElementRef;
 
+  imageError: boolean = false;
+
   acceptedFormats: string[] = ['.jpg', 'jpeg', '.png'];
 
   constructor(private renderer: Renderer2) { }
 
   fileInputchange(event) {
-    this.cleanPreviews();
-    this.createPreviewImages(event);
-    this.files.emit(event.target.files);
+    if (this.isValidImage(event)) {
```

Comment:
Otra manera de hacerlo podria ser:

this.imageError = !!event.target.files.filter(f => !f.type.includes('image/')).length;
if (!this.imageError)) { ...bla bla bla;}

Te ahorras crear el método extra para devolver un booleano nomas. Igual si querés mantenerlo separado lo podés hacer, pero es una sola linea.

Reply:
Roger that!

Reply:
Muy buen approach! por el momento lo mantengo separado